src/utils/remove-numbers/index.ts

```diff
@@ -2,6 +2,14 @@ import  { GRID } from '../../typings'
 import global  from '../../global'
 import { getRandomIndex, solveGrid, copyGrid } from '../../utils'
 
+
+
+/**
+ * Removes numbers from a full grid to create a Sudoku Puzzle.
+ * @param grid 9X9 Sudoku Grid
+ * @param attempts number of attempts to solve (higher means more difficult) - default 5
+ */
+
```

Comment:
Very nice. You should always try to document as much as you can so that other developers understand what each of your functions an components are meant to do ;)